lib/devise/models/authenticatable.rb

```diff
@@ -187,8 +191,9 @@ def devise_mailer
       def send_devise_notification(notification, *args)
         message = devise_mailer.send(notification, self, *args)
         # Remove once we move to Rails 4.2+ only.
+
         if message.respond_to?(:deliver_now)
```

Comment:
@HexterCH what do you think of this? I guess that's the only pending piece we have here.